name,energy,seconds,has_error,error_msg
##[group]GITHUB_TOKEN Permissions,192.64381082975748,2805630,False,
##[group]changed-files,183.7844574941424,1263472,False,
##[group]Run python3 -m pip install -r .tools/base_requirements.txt,139.31603972718037,12380569,False,
##[group]Fetching the repository,79.16868938209211,719977,False,
##[group]Setting up auth,20.92315362241006,141070,False,
##[group]Run actions/checkout@v4,19.792172345523028,178660,False,
##[group]Run actions/setup-python@v3,15.645240996937254,192415,False,
##[group]Checking out the ref,12.25229716627616,149798,False,
##[group]Setting up sparse checkout,12.063800286794988,93928,False,
##[group]Removing previously created refs_ to avoid conflicts,10.17883149198327,48580,False,
##[group]Run yamllint --format standard -c .tools/validation/.yamllint.yaml .,3.3304263006803,467718,True,##[error]Process completed with exit code 127.
Requested labels: self-hosted,0.0,3958554,False,
